lib/Test2/API.pm

```diff
@@ -110,7 +110,7 @@ use base 'Exporter';
 # This will make sure that Context.pm is loaded the first time this module is
 # imported, then the regular import method is swapped into place.
 sub import {
-    require Test2::API::Context unless $INC{'Test2/API/Context.pm'};
```

Comment:
The sub replaces itself once it is run.  The require and %INC check will only ever happen once per process.